### PR TITLE
DOC-5579: Added steps for getting the UXTO library

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by blsemo — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/getting-utxo-library.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+section_menu: corda5
+---
+
+# Getting the UTXO Library
+
+The Corda 5 Advanced UTXO Ledger Extensions library is an external library and not part of Corda 5. It needs to be sourced
+from Maven Central Repository or similar.
+
+1. Ensure that the repositories block of the project has `mavenCentral()` declared in the following way:
+
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/getting-utxo-library.md:20]: Should the code blocks all be indented by three spaces?

Comment (yarivus) [reply content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/getting-utxo-library.md:20]: Why 3 spaces ... either 2, 4, or 8 right ?

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/getting-utxo-library.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.0/developing-applications/api/ledger/utxo-ledger/utxo-advanced-ledger-extensions/getting-utxo-library.md:20]: indented

**Review by yarivus — APPROVED** [review]
LGTM

**Review by Niamh25 — APPROVED** [review]

**Pull request merged**